index.css

```diff
@@ -521,21 +522,13 @@ section ul, section li {
     }
     
     h4 a {
-        font-size: 18px;
-        word-wrap: break-all;
+        word-break: break-all;
```

Comment:
Okay. Until then, this PR should have something to mitigate the small screen scrolling issue. Probably the `hyphens: auto` that @benrimmington suggested. (AFAICT, the horizontal scrolling issue isn't present on `master`, so we shouldn't add it.)